src/shared/services/document-management/document-management.service.ts

```diff
@@ -26,4 +31,40 @@ export class DocumentManagementService {
         map(response => response.json())
       );
   }
+
+  createMediaViewer(caseField: CaseField): string {
+    let mediaViewer = {};
+    if (caseField.value) {
+      mediaViewer = {
+        document_binary_url: this.transformDocumentUrl(caseField.value.document_binary_url),
+        document_filename: caseField.value.document_filename,
+        content_type: this.getContentType(caseField),
+      }
+    }
+    return JSON.stringify(mediaViewer);
+  }
+
+  getContentType(caseField: CaseField): string {
+    let fileExtension = '';
+    if (caseField.value && caseField.value.document_filename) {
+      fileExtension = caseField.value.document_filename
+        .slice(caseField.value.document_filename.lastIndexOf('.') + 1);
+    }
+    if (this.isImage(fileExtension)) {
+      return DocumentManagementService.IMAGE;
+    } else if (fileExtension === 'pdf') {
+      return DocumentManagementService.PDF;
+    } else {
+      return null;
```

Comment:
shall we consider adding some sort of info/warn logging when the content type is unknown?

Reply:
Done.